CHANGELOG.md

```diff
@@ -1,13 +1,22 @@
 # Changelog
 
+## 3.0.9
+
+Added the following PostgreSQL builtin types:
+- Geometric types `line`, `lseg`,`path`,`polygon`,`box`,`circle`
+- Range types `int4range`, `int8range`, `daterange`, `tsrange`,`tstzrange`
+- Time type `time`
+
 ## 3.0.8
 
 - Properly react to connection losses by reporting the database connection as
   closed.
 
 ## 3.0.7
 
-- Allow cleartext passwords when secure connection is used. ([#283](https://github.com/isoos/postgresql-dart/pull/283) by [simolus3](https://github.com/simolus3))
+- Allow cleartext passwords when secure connection is
```

Comment:
Are these some kind of auto-formatting of the markdown file? I wouldn't mind that outside of this PR doing only just that formatting, but I'd prefer that this PR reverts the rest of the changes here.

Reply:
I added a comment explaining the constructor and made switch statement shorter. I'm unfamiliar with bit operations so I would prefer to leave it as is for now.